qiime2/sdk/usage.py

```diff
@@ -256,7 +615,24 @@ def merge_metadata(self, ref, *records):
     def _merge_metadata_(self, ref, records):
         raise NotImplementedError
 
-    def get_metadata_column(self, column_name, record):
+    def get_metadata_column(
+        self, column_name: str, record: ScopeRecord
+    ) -> 'ScopeRecord':
+        """
+        Create a new record for a specific column in example Metadata.
```

Comment:
Same comment as above - please adopt language consistent with `init_data` et al.

Reply:
Is 4e326c5 better?  I think my languaging faculties are failing me at the end of the day.